.github/workflows/hatch-and-pytest.yml

```diff
@@ -5,14 +5,14 @@ name: Build and Pytest
 
 on:
   push:
-    branches: [ "main", "release-0.2.0", "cat-devel", "mwp-devel", "pedro-devel" ]
+    branches: [ "main", "release-0.1.0", "cat-devel", "mwp-devel", "pedro-devel", "evan-devel" ]
   pull_request:
-    branches: [ "main", "release-0.2.0", "cat-devel", "mwp-devel", "pedro-devel" ]
+    branches: [ "main", "release-0.1.0", "cat-devel", "mwp-devel", "pedro-devel", "evan-devel" ]
```

Comment:
Prefer wildcards here, e.g. `"main*"` and `"release-*"`; would also resolve the conflict